### PR TITLE
The setting of ID's 

### PR DESCRIPTION
**Pull request by rubenvdlinde**
And some othere fixes from cleanup

### CODE REVIEW

File: api/docker/php/docker-entrypoint.sh [hunk 1]
```diff
@@ -38,6 +38,11 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 		echo "Updating the database"
 		bin/console doctrine:schema:update --force --no-interaction
 
+		echo "Updating commongateway plugins"
+		bin/console commongateway:composer:update
+
+		echo "Initializing the gateway"
+		bin/console commongateway:initialize
```
Comment (rjzondervan) [inline api/docker/php/docker-entrypoint.sh:45]: Isn't this double? Initialize will also update the plugins as far as I can see.

Comment (rubenvdlinde) [reply api/docker/php/docker-entrypoint.sh]: It is will change

**Review by rjzondervan — APPROVED** [review]

File: api/src/Entity/Endpoint.php [hunk 2]
```diff
@@ -309,10 +309,9 @@ public function __construct(?Entity $entity = null, ?string $customPath = null,
                 ->orderBy(['date_created' => Criteria::DESC]);
 
             // Add prefix to path
-            // todo: temp, disable prefixes for kiss
-//            if (!$entity->getCollections()->isEmpty() && $entity->getCollections()->matching($criteria)->first()->getPrefix()) {
-//                $path = $entity->getCollections()->matching($criteria)->first()->getPrefix().$path;
-//            }
+            if (!$entity->getCollections()->isEmpty() && $entity->getCollections()->matching($criteria)->first()->getPrefix()) {
```
Comment (WilcoLouwerse) [inline api/src/Entity/Endpoint.php:312]: Deze change willen we niet naar version-kiss. want dan komen all prefixes weer terug.
Dus nog even uitgecomment laten

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: api/src/Entity/Endpoint.php (hunk 2 above)
Comment (rubenvdlinde) [reply api/src/Entity/Endpoint.php]: Aangepast

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**